javascript-data-model/order-history.js

```diff
@@ -0,0 +1,43 @@
+const orderHistory = [
+  {
+    placedDate: 'August 4, 2020',
+    deliveredDate: 'August 8, 2020',
+    orderID: '114-3941689-8772232',
+    itemName: 'JavaScript for impatient programmers',
+    itemType: 'book',
+    price: 31.95,
+  },
+  {
+    placedDate: 'July 19, 2020',
+    deliveredDate: 'July 20, 2020',
+    orderID: '113-9984268-1280257',
+    itemName: 'The Timeless Way of Building',
+    itemType: 'book',
+    price: 41.33,
+  },
+  {
+    placedDate: 'July 4, 2020',
+    deliveredDate: 'July 7, 2020',
+    orderID: '114-2875557-9059409',
+    itemName: 'Gamecube Controller Adapter...',
+    itemType: 'gaming accessories',
+    price: 15.98,
+  },
+  {
+    placedDate: 'July 3, 2020',
+    deliveredDate: 'July 5, 2020',
+    orderID: '113-2883177-2648248',
+    itemName:
+      'Gamecube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+    itemType: 'gaming accessories',
+    price: 94.95,
+  },
+  {
+    placedDate: 'July 3, 2020',
+    deliveredDate: 'July 5, 2020',
+    orderID: '113-2883177-2648248',
+    itemName: 'The Art of Sql',
+    itemType: 'book',
+    price: 33.99,
+  },
```

Comment:
These are apart of the same order, so they shouldn't be 2 different object. These should be combined somehow. This is where an array would probably come in handy. You have 1 object for the whole order, but an array of all the products, so you can have multiple products apart of the same order.

Reply:
Thank you Shawn. I updated the code to have arrays in the itemName, itemType, and price properties.